2_dataCleaning/DataCleaning_Code.Rmd

```diff
@@ -262,7 +262,10 @@ You may also want to recalculate or rescale some variables. For example, we can
 mydata <- mydata %>%
   mutate(NeuroticismZ = scale(Neuroticism, center = TRUE, scale = TRUE))
 
-hist(NeuroticismZ)
+# Scale creates a matrix with width 1, sometimes this messes things up so we'll convert it to numeric          
+mydata$NeuroticismZ <- as.numeric(mydata$NeuroticismZ)
```

Comment:
can always coerce `NeuroticismZ` to numeric type inside of the first `mutate()` call